students/junghoon/westagram/urls.py

```diff
@@ -1,4 +1,5 @@
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
-]
+    path('/user', include('user.urls')),
```

Comment:
1. 제거 이 부분에 잘 못된 가이드를 드렸습니다. 제가 알려드린데로 코드를 수정을 하고 테스트 진행 해보셨나요? 아마 진행이 안 되었을 것 같은데 확인해 주세요.
2. user 보다는 users로 해주시면 좋을 것 같습니다. 정확한 이유는 나중에 RESTful API에 대해 배우실텐데 그 전에 이유가 궁금하시면 라운지 시간에 질문해 주세요.

```suggestion
    path('users', include('user.urls')),
```

Reply:
1. http://127.0.0.1:8000//user/signup으로 접속해야 했습니다.
2. 넵! 배우고 이해가 안가면 질문하겠습니다.